### PR TITLE
Adds guides to make debug cables and open Kuri's torso

### PR DESCRIPTION
**Pull request by KaijenHsiao**
These guides are to make it easier to reply to people who are having issues with their Kuris.
I may add pictures later, but don't have time now, figure it's better to have the guides in with no pictures than no guides.

90% feedback appreciated.

### CODE REVIEW

File: guides/hardware/debug-cable.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+(If you don't want to use sudo, add your username to the `dialout` group
+in /etc/groups.)
+
+## Talking to Kuri from Windows
+If you are using a Windows computer,
+download [PuTTY](https://www.putty.org/), switch the Connection Type to Serial,
+enter the correct
+COM port under Serial Line, edit Speed to use a baud rate of 115200, and
+the use the Open button to start the session.
+
```
Comment (cjwilliams) [inline guides/hardware/debug-cable.md]: Instructions for OSX? Should be similar, but it'll be something like `/dev/cu.usb*` instead

File: guides/hardware/opening-up-kuri.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Goal
+Remove Kuri's front shell so that you can get at her main board.
+
+## Overview
+To get at the inside of Kuri's torso, we will remove both arm plates, unscrew
+the screws underneath, then pop the front shells off and unhook the cables
+connecting the shells to the rest of Kuri.
+
+## Removing Kuri's arm plates
```
Comment (cjwilliams) [inline guides/hardware/opening-up-kuri.md]: Any chance we could add some photos or drawings to these steps? Maybe it'd be worthwhile to PNG-ize and embed one of the subassembly drawings here

File: guides/hardware/opening-up-kuri.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+front.  Also remove the screw holding the speaker grill on, then slide
+the speaker grill upwards to remove it.  Now unscrew the screw on the side
+closest to Kuri's front that was underneath the speaker grill.
+
+## Removing Kuri's front torso shells
+Kuri's front torso shells (both the white top and the black bottom shells) are
+only held on by snap-fit.  Put Kuri on the ground, facing you, and grab
+both top and bottom shells at once, on both sides.  Pull outwards and towards
+you, wiggling (and deforming slightly) the plates to release all the
+snap-fit attachments.  Be careful with the connecting cables as you pull
```
Comment (cjwilliams) [inline guides/hardware/opening-up-kuri.md]: s/connecting/connected

Maybe quote this line so it looks like
> WARNING: Be careful with the connected cables as you pull the shells off

File: guides/hardware/opening-up-kuri.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+cable to the depth sensor goes through a cable-guide loop.
+
+## Main board connections
+Now that Kuri's main board is exposed, there are two easy connection points
+of interest: one is the serial debug port, which you can use with a
+custom serial cable (see the ``Making and using a debug cable`` guide).
+The other is the open USB 3.0 port, which you can use to hook up USB devices
+to Kuri's main computer.
+
+## Removing Kuri's rear torso shells
```
Comment (cjwilliams) [inline guides/hardware/opening-up-kuri.md]: Same comment about a stylized warning

File: guides/hardware/debug-cable.md (hunk 1 above)
Comment (KaijenHsiao) [reply guides/hardware/debug-cable.md]: I don't have a Mac, so can't add instructions for it.  You could add them, if you know how, or we could just leave it--I don't feel like it has to cover every possible option. :P

File: guides/hardware/opening-up-kuri.md (hunk 2 above)
Comment (KaijenHsiao) [reply guides/hardware/opening-up-kuri.md]: I can add photos later this week or next week, but as people are trying to onboard their Kuris as we speak and many of them are failing at it for the exact same reason, I would like to get these two PRs merged ASAP so I can email them back with instructions that are actually in the documentation.  Or I can take out the reference to this one, and just merge the other.

**Pull request merged**